frontend/app/src/components/display/AccountDisplay.vue

```diff
@@ -45,14 +56,25 @@ export default defineComponent({
     const { account } = toRefs(props);
     const { scrambleData, shouldShowAmount } = setupDisplayData();
 
+    const { ensNameSelector } = setupEnsNames();
```

Comment:
not important right now but do you think it would be better to change the `setup` to `use` in the future to follow the scheme suggested with other composables?